Fixes memory leak

ploverlake opened the pull request:
The std::vector won’t delete the objects that are pointed to.

ploverlake committed:
Fixes memory leak

jmillan commented on src/sdp/RemoteSdp.cpp:
```diff
@@ -76,6 +76,15 @@ namespace mediasoupclient
 
+	Sdp::RemoteSdp::~RemoteSdp()
+	{
+		MSC_TRACE();
```
```suggestion
		MSC_TRACE();
		
```

jmillan commented:
Great,

Just a cosmetic change requested.

ploverlake committed:
Fixes memory leak

We MUST take the ownership of |desc| in callback 'CreateSessionDescriptionObserver::OnSuccess'.

ploverlake commented:
I have pushed one more commit to fix another memory leak.

See the comment of CreateSessionDescriptionObserver::OnSuccess in /path/to/webrtc/src/api/jsep.h, the the ownership of the |desc| is transferred by the callback, so we MUST take the ownership.

ploverlake committed:
Update src/sdp/RemoteSdp.cpp

Co-authored-by: José Luis Millán <jmillan@aliax.net>

jmillan commented on src/PeerConnection.cpp:
```diff
@@ -444,9 +444,12 @@ namespace mediasoupclient
 	{
 		MSC_TRACE();
 
+		// This callback should take the ownership of |desc|.
```
Please use camelCase.

ploverlake replied:
Sorry, I'll do it.

ploverlake committed:
Use camelCase

jmillan commented:
Thanks @ploverlake!
